javav2/example_code/kinesis/src/main/java/com/example/kinesis/KinesisStreamEx.java

```diff
@@ -86,7 +76,7 @@ public static void main(String[] args) {
     // snippet-end:[kinesis.java2.stream_example.lifecycle_callback]
 
     /**
-     * Uses the SubscribeToShardResponseHandler.Builder and a simple consumer of events to subscribe
+     * Using the SubscribeToShardResponseHandler.Builder and a simple Consumer of events to subscribe.

```

Comment:
**Uses** the...
(for consistency, because all the other comments are in this tense)

Reply:
fixed